cypress/page/products-list.page.ts

```diff
@@ -0,0 +1,21 @@
+class ProductListPage {
+  private addToCartBtn: string;
+  private proceedToCheckoutBtn: string;
+
+  constructor() {
+    this.addToCartBtn =
+      "#center_column a.button.ajax_add_to_cart_button.btn.btn-default";
+    this.proceedToCheckoutBtn =
+      "[style*='display: block;'] .button-container > a";
+  }
+
+  public clickAddToCartBtn(): void {
+    cy.get(this.addToCartBtn).click();
+  }
+
+  public clickProceedToCheckout(): void {
+    cy.get(this.proceedToCheckoutBtn).click();
+  }
+}
+
+export default new ProductListPage();
```

Comment:
not sure about this approach, I prefer the one suggested by the guide, export the class and then have a page/index.ts so you can import from the page index in the buy-shirt.cy.ts file